app/models/article.rb

```diff
@@ -388,6 +388,16 @@ def self.unique_url_error
 
   scope :eager_load_serialized_data, -> { includes(:user, :organization, :tags) }
 
+  scope :above_average, lambda {
+    order(:score).where("score >= ?", average_score)
```

Comment:
Note sure where to put this, but potentially note-worthy: the way this is being managed by ActiveRecord, `average_score` will wind up inheriting the bounding scope (ie, if we're scoped to `followed tags`, `average_score` will be calculated _based on articles scoped to those tags_). I _think_ this is more-or-less fine (?), but I'm going to ping the original issue to clarify.

Reply:
I _think_ this is the expected product behaviour? Or at least the issue mentions that organizations returned should (ideally) be "organizations that are producing recent, at least average quality, content under tags that the user has followed".

However there's also the caveat that there still has to be at least 5 organizations returned, so this might need to be combined/`UNION`-ed with an unscoped query to ensure that there are enough results